tests/golden/test_runner.py

```
@@ -52,25 +53,36 @@ def populated_cache(tmp_path):
 
-    def fake_execute(work_dir, tensors, **kwargs):
-        if write_outputs_positional is None:
-            return
-        for tensor, value in zip(tensors, write_outputs_positional):
-            if value is not None:
-                tensor[:] = value
+    if fake_execute is None:
+        def fake_execute(work_dir, tensors, **kwargs):
+            if write_outputs_positional is None:
+                return
+            for tensor, value in zip(tensors, write_outputs_positional):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)